bh/documentation/src/colors/examples/Contrast.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="bh-grid">
+    <div class="bh-grid__item">
+      <div class="contrast-colors__test">
```

Comment:
this class should be called 'test'? -> contrast-colors__test

Reply:
yes, I create two tests, asc and desc colors